src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -25,47 +25,115 @@
+  private static final String FORMAT_DOC = "The format class to use when writing data to HDFS.";
+  public static final String FORMAT_DEFAULT = "io.confluent.connect.hdfs.avro.AvroFormat";
+
   public static final String FLUSH_SIZE_CONFIG = "flush.size";
   private static final String FLUSH_SIZE_DOC = "The number of records needed to rotate files.";
+
   public static final String ROTATE_INTERVAL_CONFIG = "rotate.interval.ms";
   private static final String ROTATE_INTERVAL_DOC = "The interval to rotate files.";
-  public static final String TOPIC_DIR_CONFIG = "topic.dir";
-  private static final String TOPIC_DIR_DOC = "HDFS directory to store data.";
-  public static final String TOPIC_DIR_DEFAULT = "topics";
+
+  public static final String TOPICS_DIR_CONFIG = "topics.dir";
+  private static final String TOPICS_DIR_DOC = "HDFS directory to store data.";
+  public static final String TOPICS_DIR_DEFAULT = "topics";
+
+  public static final String LOGS_DIR_CONFIG = "logs.dir";
+  private static final String LOGS_DIR_DOC = "HDFS directory to store logs.";
+  public static final String LOGS_DIR_DEFAULT = "logs";
+
   public static final String SCHEMA_CACHE_SIZE_CONFIG = "schema.cache.size";
-  public static final int SCHEMA_CACHE_SIZE_DEFAULT = 1000;
   private static final String SCHEMA_CACHE_SIZE_DOC = "The size of the schema cache.";
+  public static final int SCHEMA_CACHE_SIZE_DEFAULT = 1000;
+
   public static final String STORAGE_CLASS_CONFIG = "storage.class";
   private static final String STORAGE_CLASS_DOC = "The storage layer.";
-  public static final String STORAGE_CLASS_DEFAULT = "io.confluent.connect.hdfs.HdfsStorage";
+  public static final String STORAGE_CLASS_DEFAULT = "io.confluent.connect.hdfs.storage.HdfsStorage";
+
   public static final String RETRY_BACKOFF_CONFIG = "retry.backoff.ms";
   private static final String RETRY_BACKOFF_DOC = "The retry backoff in milliseconds.";
-  public static final long RETRY_BACKOFF_DEFALUT = 5000L;
+  public static final long RETRY_BACKOFF_DEFAULT = 5000L;
+
   public static final String HIVE_HOME_CONFIG = "hive.home";
   private static final String HIVE_HOME_DOC = "Hive home directory";
+  public static final String HIVE_HOME_DEFAULT = "";
+
   public static final String HIVE_CONF_DIR_CONFIG = "hive.conf.dir";
   private static final String HIVE_CONF_DIR_DOC = "Hive configuration directory";
+  public static final String HIVE_CONF_DIR_DEFAULT = "";
+
+  public static final String HIVE_INTEGRATION_CONFIG = "hive.integration";
+  private static final String HIVE_INTEGRAtiON_DOC = "Integrate with Hive";
+  public static final boolean HIVE_INTEGRATION_DEFAULT = false;
+
+  public static final String HIVE_DATABASE_CONFIG = "hive.database";
+  private static final String HIVE_DATABASE_DOC = "The database to use when writing to hive.";
+  private static final String HIVE_DATABASE_DEFAULT = "default";
+
+  public static final String AWAIT_TERMINATION_TIMEOUT_CONFIG = "hive.executor.shutdown.timeout.ms";
```

Comment:
This naming is very specific and exposes internal implementation details. What about just having an overall shutdown timeout?